python_modules/dagster/dagster/_core/definitions/asset_sla.py

```diff
@@ -0,0 +1,46 @@
+import pendulum
+from croniter import croniter
+
+
+class AssetSLA:
+    def is_passing(self, data_timestamp: float):
+        pass
+
+
+class StalenessSLA(AssetSLA):
+    """This SLA specifies that an asset must not be more than `allowed_staleness` time out of date
```

Comment:
It would be very helpful (for users, but also for review) to have a full, clear description here.